stores/sql/sqlite/migrations/main/migration_00019_scan_reset.sql

```diff
@@ -0,0 +1 @@
+UPDATE hosts SET settings = NULL, price_table = NULL, last_scan = NULL, last_scan_success = NULL, second_to_last_scan_success = NULL WHERE 1=1;
```

Comment:
Is settings = NULL okay? The default `INSERT hosts` looks like it does an empty table.

Reply:
Good question. Haven't had time to test it yet